boards/imix/src/main.rs

```diff
@@ -327,8 +326,9 @@ pub unsafe fn reset_handler() {
          &sam4l::gpio::PC[28], // P5
          &sam4l::gpio::PC[27], // P6
          &sam4l::gpio::PC[26], // P7
-         &sam4l::gpio::PC[25], // P8
-         &sam4l::gpio::PC[25]], // Dummy Pin (regular GPIO)
+         &sam4l::gpio::PA[20], // P8
+         &sam4l::gpio::PA[20], // Dummy Pin (regular GPIO)
```

Comment:
Yeah so the dummy pin was here originally, and I didn't know if it actually served a purpose or if it was just some cruft. If there's no reason for it, I can just take it out.